pkg/cli/cvd.go

```diff
@@ -358,24 +357,43 @@ func credentialsFactoryFromSource(source string) (CredentialsFactory, error) {
 	case InjectedCredentialsSource:
 		return func() string { return client.InjectedCredentials }, nil
 	default:
-		return jwtAccessToken(source)
+		// expected: `(jwt|oauth):/dir/credentialFile`
+		return accessToken(source)
 	}
 }
 
-func jwtAccessToken(filepath string) (func() string, error) {
+const (
+	jwtAuthType   = "jwt"
+	oauthAuthType = "oauth"
+)
+
+func accessToken(source string) (func() string, error) {
+	strs := strings.SplitN(source, ":", 2)
+	if strs == nil || len(strs) != 2 {
+		return nil, errors.New("unknown credential type, only accept: `none`/`injected`/`jwt:'<filepath>'`/`oauth:'<filepath>'`")
```

Comment:
nit: In the message, what about not mentioning jwt or oauth? I think we can replace it into `'<authType>':'<filepath>'`?

Reply:
May I know the reason why you prefer `'<authType>':'<filepath>'`?

In my opinion, it might be slightly confusing for users who are unfamiliar with the possible values of <authType>.

Reply:
+1 to @chyijia's concern. It's good to show the supported auth types in our error message. However, I think in most of the cases when an user sets a wrong auth type, our code would probably return error at line#395 (which already prints all the supported auth types). So whether or not to print it here, I think either way is fine.

Also, we should use `jwtAuthType` and `oauthAuthType` to format the error message. This is to prevent from modifying the raw strings at multiple places if we need to change the auth types in the future.

Reply:
A possible failure occurs when `--credentials_source=/dir/file` is used. An error message might appear on line#373, and users may find it difficult to determine the appropriate `authType`.

Reply:
The reason for preferring `'<authType>':'<filepath>'` was to avoid hard-coded value or mentioning supported auth type here. Since we're providing duplicated information below. But I can understand your concern... User doesn't know enough information until trying to set arbitrary `authType` with proper `filepath`.

Alternative option would be adding the help message of `cvdr create`, but not certain about the best way considering UX. It's okay to ignore this comment.